src/App.tsx

```diff
@@ -1,26 +1,75 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { ErrorTypes, FilterTypes } from './types/enums';
+import { handleError, prepareVisibleTodos } from './utils/services';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<number[]>([]);
```

Comment:
It's not a boolean variable, as previous reviewer mentioned `loadingTodoIds` describes better